src/Components/reservations/NewReservation.jsx

```diff
@@ -58,23 +60,17 @@ export default function NewReservation() {
   return (
     <div className="reservation-page">
 
-      <img
-        className="reservation-page-background"
-        alt="reservarion-background"
-        src={bike?.image || 'https://cdn.shopify.com/s/files/1/1439/6088/files/thin.jpg?width=100;height:100'}
-      />
+      <MirrorCover bike={bike} />
 
-      <div className="reservation-page-bg-cover" />
-
-      <h2><strong>NEW RESERVATION</strong></h2>
+      <h2 className="auth-title reservation-title">BOOK A TEST-RIDE</h2>
```

Comment:
also can you add the view or just in this same component let user select location first then move them to the form to select duration as shown in the design 
![Capture d’écran du 2023-11-13 14-31-58](https://github.com/khanh-devos/capstone-ebike-reservation-frontend/assets/59707859/f184a1a3-5dde-4156-a103-129dc671ed41)